addons/nametags/stringtable.xml

```diff
@@ -145,6 +145,21 @@
             <Chinesesimp>预设名字颜色 (非同小队队友) </Chinesesimp>
             <Chinese>預設名稱顏色 (非同小隊隊友) </Chinese>
         </Key>
+        <Key ID="STR_ACE_NameTags_MainNametagColor">
+            <English>Default Team Name Tag Color</English>
```

Comment:
All these strings could be moved to settings subcategory like "Team Tag colors" and each setting could be named like `Red` or `Red team` and strings for them could be found in vanilla strings.

Reply:
you mean like ace logistics settings has multiple subcategories? if I'm looking at this correctly that would require adding those settings in SQF instead of config (because category is assumed to be text, not an array).

The colours seem to be available in vanilla strings in
- str_a3_texturesources_red0
- str_a3_texturesources_blue0
- str_a3_texturesources_green0
- str_a3_texturesources_yellow0

I was unable to find ones that said `team` as well.
Is it ok to use those? They don't seem to be intended for this kind of use but I suppose they'd do.
Would copying them to ACE strings be better?

Reply:
Yes I mean those subcategories and yes this will require SQF settings not config.
I don't know if there are suitable strings (out of arma ATM), I meant those used in `9` (`Assign team`) vanilla action menu.
I think any strings with colors suit and no need to copy them to ACE.

Reply:
alright, I'll look into it